build.gradle

```diff
@@ -117,8 +117,8 @@ configure(allprojects) {
 		"-Xlint:rawtypes",
 		"-Xlint:deprecation",
 		"-Xlint:unchecked",
-		"-Xlint:-options",
-		"-Werror"
+		"-Xlint:-options"
+//		"-Werror"
```

Comment:
Had the same problem when upgrading to the new CFJavaClient, kind of annoying that we have so many warnings. That is something to check with them.